autoscheduler/frontend/src/redux/actions/termData.ts

```diff
@@ -118,7 +119,12 @@ export interface UpdateSortTypeAction {
   sortIsDescending: boolean;
 }
 
+export interface ScrollSectionIntoViewAction {
+  type: 'SCROLL_SECTION_INTO_VIEW';
+  section: Section;
+}
+
```

Comment:
This action actually just expands a card if it exists, or shows an error if it doesn't and sets it to section view, correct? The current name is misleading since no scrolling actually occurs here

Reply:
Yeah this is a great point, was really just left over from my initial plans of how this would work